### PR TITLE
TAP-4176:fix unwind error

### PR DESCRIPTION
**Pull request by weiliang110100**

### CODE REVIEW

**Comment by xbsura** [comment]
SonarQube Quality Gate Status: **ERROR**

- Status: **ERROR**, MetricKey: **new_coverage**, ActualValue: **0.0**


See Sonar Scan Result at: http://58.251.34.123:29000/dashboard?branch=fix_TAP-4176_unwind_error&id=tapdata

File: iengine/iengine-app/src/main/java/io/tapdata/flow/engine/V2/node/hazelcast/processor/HazelcastProcessorBaseNode.java [hunk 1]
```diff
@@ -374,8 +374,8 @@ protected void tryProcess(List<BatchEventWrapper> tapdataEvents, Consumer<List<B
 						processResult = getProcessResult(TapEventUtil.getTableId(tapdataEvent.getTapEvent()));
 					}
 				}
-				batchEventWrapper.setTapdataEvent(event);
-				BatchProcessResult batchProcessResult = new BatchProcessResult(batchEventWrapper, processResult);
+				BatchEventWrapper finalBatchEventWrapper = new BatchEventWrapper(event,batchEventWrapper.getProcessAspect());
```
Comment (ply0011) [inline iengine/iengine-app/src/main/java/io/tapdata/flow/engine/V2/node/hazelcast/processor/HazelcastProcessorBaseNode.java:377]: 丢失了原先batchEventWrapper的一些属性

**Review by ply0011 — CHANGES_REQUESTED** [review]

**Comment by xbsura** [comment]
SonarQube Quality Gate Status: **ERROR**

- Status: **ERROR**, MetricKey: **new_coverage**, ActualValue: **66.66666666666667**


See Sonar Scan Result at: http://58.251.34.123:29000/dashboard?branch=fix_TAP-4176_unwind_error&id=tapdata

**Review by ply0011 — CHANGES_REQUESTED** [review]
1. 根据处理节点实现，来控制是否需要clone
2. 处理与分支TAP-4266的冲突

**Review by ply0011 — APPROVED** [review]

**Pull request merged**